Tests/MongoSwiftTests/CrudTests.swift

```diff
@@ -228,7 +226,93 @@ private class AggregateTest: CrudTest {
 
 private class BulkWriteTest: CrudTest {
     override func execute(usingCollection coll: MongoCollection<Document>) throws {
-        XCTFail("Unimplemented")
+        let requestDocuments: [Document] = try self.args.get("requests")
+        var requests = [WriteModel]()
+
+        for request in requestDocuments {
```

Comment:
nit: you could use `map` to one-line initializing and filling in `requests`

Reply:
Refactored this based on https://useyourloaf.com/blog/swift-guide-to-map-filter-reduce/.